<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileUploadWin</class>
 <widget class="QWidget" name="ProfileUploadWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="MainBoard" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <widget class="QScrollArea" name="ProfileArea">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>152</width>
      <height>152</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="Size50">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>50*50</string>
    </property>
   </widget>
   <widget class="QLabel" name="Size50Img">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Size100">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>100*100</string>
    </property>
   </widget>
   <widget class="QLabel" name="Size100Img">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Size150">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>150*150</string>
    </property>
   </widget>
   <widget class="QLabel" name="Size150Img">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>91</width>
      <height>112</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="ChooseFile">
       <property name="text">
        <string>选择文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ReadImg">
       <property name="text">
        <string>查看效果</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Upload">
       <property name="text">
        <string>上传</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Close">
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
